### ISSUE TITLE
Java classes not generated

### ISSUE THREAD

**Issue by divvy123** [aoudiamoncef/apollo-client-maven-plugin#58]
To generate classes in Java.
I created a query named SampleTest in src/main/graphql/SampleTest.graphql
Then added schema.json in the same directory.
when I'm doing mvn compile or mvn generate-sources. The build is successful but doesnt generate classes.

I have placed the plugin thus:
`<build>
	    <pluginManagement>
		<plugins>
		    <plugin>
		        <groupId>com.github.aoudiamoncef</groupId>
		        <artifactId>apollo-client-maven-plugin</artifactId>
		        <version>4.0.5</version>`

**Comment by aoudiamoncef** [comment]
Hi, could you share your complete plugin configuration ? 

Java version 
Maven version

**Comment by divvy123** [comment]
The `<pluginManagement>` caused the issue.  I had added that tag as i use Eclipse.
I removed that tag and let it  show the error(just ignored) on Eclipse in pom file.
From the command prompt i was able to execute mvn generate-sources and it created the java files.
Thank you so much !


Eclipse shows  error in the pom.xl at <execution>
Plugin execution not covered by lifecycle configuration: com.github.aoudiamoncef:apollo-client-maven-plugin:4.0.5:generate (execution: 
 default, phase: generate-sources)
